### PR TITLE
Update extension mangakawaii after the new version of Website

### PR DESCRIPTION
**Pull request by Angelghost**

### CODE REVIEW

File: src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt [hunk 1]
```diff
@@ -96,17 +99,20 @@ class MangaKawaii : ParsedHttpSource() {
...
-        val match = regex.findAll(element).map { it.value.substringAfter("data-src\", \" ").substringBefore("\"").trim() }
-        // throw Exception(match.elementAt(1))
+        var div = body.select("div#all")
+       var elements= div.select("img")
+
         val pages = mutableListOf<Page>()
-        for (i in 0 until match.count()) {
-            pages.add(Page(i, "", match.elementAt(i)))
+        for (i in 0 until elements.count()) {
+            Log.i(TAG, elements[i].attr("data-src").trim())
```
Comment (arkon) [inline src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt:107]: Please remove the logs.

File: src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt [hunk 2]
```diff
@@ -96,17 +99,20 @@ class MangaKawaii : ParsedHttpSource() {
 
     override fun pageListParse(response: Response): List<Page> {
         val body = response.asJsoup()
-        val element = body.select("script:containsData(Imagesrc)").toString()
-        val regex = "(data-src).*[\"]".toRegex()
-        val match = regex.findAll(element).map { it.value.substringAfter("data-src\", \" ").substringBefore("\"").trim() }
-        // throw Exception(match.elementAt(1))
+        var div = body.select("div#all")
+       var elements= div.select("img")
```
Comment (arkon) [inline src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt:103]: Nitpick: spacing

File: src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt [hunk 3]
```diff
@@ -96,17 +99,20 @@ class MangaKawaii : ParsedHttpSource() {
...
-            pages.add(Page(i, "", match.elementAt(i)))
+        for (i in 0 until elements.count()) {
+            Log.i(TAG, elements[i].attr("data-src").trim())
+            pages.add(Page(i, "", elements[i].attr("data-src").trim() ))
         }
         return pages
     }
 
+
+
```
Comment (arkon) [inline src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt:114]: Revert these empty line additions.

File: src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt [hunk 4]
```diff
@@ -28,21 +29,22 @@ class MangaKawaii : ParsedHttpSource() {
...
-    override fun popularMangaRequest(page: Int) =
-        GET("$baseUrl/filterLists?page=$page&sortBy=views&asc=false", headersBuilder().add("X-Requested-With", "XMLHttpRequest").build())
+
+    override fun popularMangaRequest(page: Int) = GET("$baseUrl/liste-manga/filterMangaList?page=$page&sortBy=views&asc=false", headersBuilder().add("X-Requested-With", "XMLHttpRequest").build())
+
     override fun latestUpdatesRequest(page: Int) = GET(baseUrl, headers)
     override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
         val uri = Uri.parse("$baseUrl/search").buildUpon()
-            .appendQueryParameter("query", query)
+            .appendQueryParameter("query", query).appendQueryParameter("search_type", "manga")
```
Comment (arkon) [inline src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt:47]: Nitpick: put it on a new line

File: src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt [hunk 5]
```diff
@@ -1,6 +1,7 @@
 package eu.kanade.tachiyomi.extension.fr.mangakawaii
 
 import android.net.Uri
+import android.util.Log
```
Comment (arkon) [inline src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt:4]: Remove this.

**Review by arkon — CHANGES_REQUESTED** [review]

File: src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt (hunk 1 above)
Comment (Angelghost) [reply src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt:107]: Too fast to check the code.. Better than at my work ...
I will change it .. it's late on my side forget to analyse my commit sorry

Comment (Angelghost) [reply src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt]: Done

File: src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt (hunk 2 above)
Comment (Angelghost) [reply src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt]: Done

File: src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt (hunk 4 above)
Comment (Angelghost) [reply src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt]: Done

File: src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt (hunk 5 above)
Comment (Angelghost) [reply src/fr/mangakawaii/src/eu/kanade/tachiyomi/extension/fr/mangakawaii/MangaKawaii.kt]: Done

**Review by arkon — APPROVED** [review]

**Pull request merged**